custom_components/tahoma/tahoma_device.py

```diff
@@ -66,11 +66,7 @@ def assumed_state(self) -> bool:
     @property
     def device_state_attributes(self) -> Dict[str, Any]:
         """Return the state attributes of the device."""
-        attr = {
-            "ui_class": self.device.ui_class,
```

Comment:
Those details are added as device information and in the debug log. Since last versions the attributes are added to the front-end, thus this is cluttering the view.